src/foam/u2/crunch/CapabilityFeatureView.js

```diff
@@ -110,18 +112,21 @@ foam.CLASS({
             'background-image': "url('" + self.data.icon + "')"
           })
           .add(this.slot(function(cjStatus) {
-            return this.E().addClass(style.myClass('tooltip'))
+            return this.E()
+            .addClass(style.myClass('tooltip'))
               .start('span')
                 .addClass(style.myClass('tooltiptext'))
                 .addClass(style.myClass('tooltip-bottom'))
                 .enableClass(style.myClass('tooltipDisabled'), self.tooltipEnabled, true)
                 .add(cjStatus.documentation)
               .end()
-              .add(cjStatus.label).addClass(style.myClass('badge'))
-              .style({
-                'background-color': cjStatus.background,
-                'color': cjStatus.color
-              });
+              .start(foam.u2.view.BadgeView, {label: cjStatus.label, color: cjStatus.color, backgroundColor: cjStatus.background, isAttachedToCard: true}).addClass(self.myClass('badge')).end()
```

Comment:
If you are using VS code as your IDE can you turn on ESLint. Our standard is generally 80char per line

Reply:
What is says is: "Line lengths should be 80 characters or less, except for embedded data, like templates or sprites, or when modifying the code to make it fit in less than 80 characters would actually makes it less readable."
Source: https://github.com/foam-framework/foam/wiki/CodingStyle

In this case, you could definitely indent between start() and end() and improve readability.